charts/kong/README.md

```diff
@@ -0,0 +1,17 @@
+Usage
+==========
+
+Firstly install helm: [docs here](https://helm.sh/)
+
+1. Clone the repository
+2. cd ./charts/kong/
+3. edit the "values.yaml" doc to fit your environment
+3. helm package .
+4. helm install kong-0.1.1.tgz --name yourreleasename
+
+You will then see notes on how to connect and use your Kong deployment. 
+
+Todo:
+--------
+
+- Fix cassandra deployment issues 
```

Comment:
@lawrencegripper What wrong with Cassandra deployment?

Reply:
@shashiranjan84 how many times have you deployed Cassandra in Kubernetes?

Reply:
@ipedrazas none with this chart, currently I am in process of updating this PR, please check https://github.com/Kong/kong-dist-kubernetes/tree/charts. I will make sure Kong works with both Postgres and Cassandra.